src/declarations/interfaces.d.ts

```diff
@@ -0,0 +1,8 @@
+import types from "./types";
+
+export declare interface ValidationFunction {
+  (name: string): types.ValidationResult;
+}
+export declare interface StepAction {
```

Comment:
can we add whitespace between the end of one interface and the start of the next?

same sort of whitespacing goes for in the `types` file.